### PR TITLE
ZA2-Kabelo Letlala-JavaScript-Core-1-Coursework-Week1

### PR DESCRIPTION
**Pull request by M3ntalist**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Kabelo
- Your City: Cape Town
- Your Slack Name: Kabelo Letlala AKA M3ntalist

# Homework Details

- Module: JavaScript-Core-1-Coursework
- Week: 1

# Notes

- What did you find easy? Exercises where great

- What did you find hard? Mandatory coursework especially Extra course-work

- What do you still not understand? I need to learn and work more on Functions

- Any other notes? When I test my code on the terminal it returns failed, showing that there are some errors on "======= TESTS - DO NOT MODIFY ====="

### CODE REVIEW

File: exercises/D-strings/exercise.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
+let message = "This is a string"
+//log
 console.log(message);
```
Comment (AnthonyMogotlane) [inline exercises/D-strings/exercise.js:4]: Below this line, you were supposed to console the data type of the variable message using typeof operator.

File: exercises/F-strings-methods/exercise2.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
 const name = " Daniel  ";
-
-console.log(message);
+let exerciseOne =
+"My name is " + name+ " and my name is " + nameLength + " characters long";
```
Comment (AnthonyMogotlane) [inline exercises/F-strings-methods/exercise2.js]: nameLength is not defined, you need to declare the nameLength variable first before you can use it on this line. You should use trim() buildin method to remove whitespace on the sides of the name.

**Review by AnthonyMogotlane — CHANGES_REQUESTED** [review]
Well done Kabelo, apart from the lines commented below, I am happy with your work. Keep it up!

File: exercises/D-strings/exercise.js (hunk 1 above)
Comment (M3ntalist) [reply exercises/D-strings/exercise.js:4]: Thanks, Anthony

File: exercises/F-strings-methods/exercise2.js (hunk 2 above)
Comment (M3ntalist) [reply exercises/F-strings-methods/exercise2.js]: Sure, you are right, somehow I deleted the right one and type the wrong code while getting frustrated with the failure of the Mandatory coursework

**Review by diegoharera — COMMENTED** [review]
Hi @kabelo , I understand string concatenation takes basic exercises like these. You did well in this exercise analyzing your code and code structure

**Pull request closed without merging**